internals/cli/cli.go

```diff
@@ -174,15 +175,13 @@ func Parser(cli *client.Client) *flags.Parser {
 	parser.ShortDescription = "Tool to interact with pebble"
 	parser.LongDescription = longPebbleDescription
 
-	// Hide the global --version option on every command
-	if version := parser.FindOptionByLongName("version"); version != nil {
-		version.Description = "Print the version and exit"
-		version.Hidden = true
-	}
-
 	// Add --help like what go-flags would do for us, but hidden
 	addHelp(parser)
 
+	// Regular expressions for positional and flag arguments
+	positionalRegexp := regexp.MustCompile(`^<[\w-]+>$`)
+	flagRegexp := regexp.MustCompile(`^-(\w|-[\w-]+)$`)
```

Comment:
This may not be an issue, but just pointing out that for flag validation, the following will pass:

"---"
"---------"
"-------gtrfffg---hhhh"